examples/search/analysis.ini

```diff
@@ -172,19 +172,22 @@ sngl-ranking = newsnr_sgveto_psdvar
 randomize-template-order =
 statistic-files = ${resolve:./statHL.hdf} ${resolve:./statLV.hdf} ${resolve:./statHV.hdf} ${resolve:./statHLV.hdf}
 
-[coinc-full_data]
+[coinc-defaultvalues]
 timeslide-interval = 0.1
 
+[coinc-full_data]
+
 [coinc-full_data-2det]
 loudest-keep-values = [-1:5,1:5]
 
 [coinc-full_data-3det]
 loudest-keep-values = [-3:5,-1:5]
+timeslide-interval = 0.11
 
 [coinc-fullinj&coinc-injfull]
-timeslide-interval = ${coinc-full_data|timeslide-interval}
 cluster-window = ${statmap|cluster-window}
 loudest-keep-values = 15.0:9999999999999
+timeslide-interval = ${coinc-defaultvalues|timeslide-interval}
```

Comment:
Wait, as a defaultvalue, why would I be specifying it here? I thought we should get the defaultvalue by default??

Reply:
Or just another illustration?? Perhaps a confusing one without comments if so.

Reply:
So this is something I'm not sure should be fixed, which is that the interpolation is done before the default values are set, so in order to use this feature, the `-defaultvalues` subsection must be specified

Reply:
But why *would* you do this. This section should get the defaultvalue anyway. Wouldn't the code work fine with this line just removed?

Reply:
However in this specific case, it is an illustration, and so is not needed (and I am adding a comment)

Reply:
(I also don't see the indicated comment)

Reply:
I will change this, I couldnt see a better example before, but found one for `fit_by_template` and `plot_binnedhist`